src/overlays/actors/ovl_En_Sw/z_en_sw.h

```diff
@@ -14,39 +14,48 @@ typedef struct EnSw {
     /* 0x0190 */ EnSwActionFunc actionFunc;
     /* 0x0194 */ ColliderJntSph collider;
     /* 0x01B4 */ ColliderJntSphElement sphs[1];
-    /* 0x01F4 */ Color_RGBA8 unk_1F4;
+    /* 0x01F4 */ Color_RGBA8 limb4Col; // never set past black.
     /* 0x01F8 */ Vec3s jointTable[30];
     /* 0x02AC */ Vec3s morphTable[30];
-    /* 0x0360 */ u8 unk_360;
-    /* 0x0364 */ Vec3f unk_364;
+    /* 0x0360 */ u8 goldHiddenBool; // set when revealed, unset when landing.
```

Comment:
```suggestion
    /* 0x0360 */ u8 goldIsHidden; // set when revealed, unset when landing.
```

Reply:
the comment contradicts the name, no? "set when revealed" means its true when its not hidden anymore? and if its indicating information about revealed vs landing then its prob broader scope than just "is hidden"

I have not reviewed the code much at all yet though, so sorry if my comment is misguided

Reply:
Oh right that is backwards (my bad). Based on the comment perhaps something like `goldInAir`, as it is used when it is revealed and gets flown in the air, until it lands.